docs/src/Guides/5 Components.md

```diff
@@ -201,10 +201,37 @@ When responding to a component you need to satisfy discord either by responding
                 await ctx.send("You clicked it!")
     ```
 
-=== ":three: Persistent Callback Option 2"
+=== ":two: Persistent Callback Option 2"
     If you have a lot of components, putting everything in the `on_component()` event can get messy really quickly.
 
-    Personally, I put my callbacks into different files. For this example to work, the function name needs to be the same as the `custom_id` of the component.
+    Similarly to Option 1, you can define `@component_callback` listeners. This works after restarts too.
+
+    You have to pass your `custom_id` to `@component_callback(custom_id)` for the libary to be able to register the callback function to the wanted component.
+
+    ```python
+    async def my_command(...):
+        components: list[ActionRow] = [
+            ActionRow(
+                Button(
+                    custom_id="my_button",
+                    style=ButtonStyles.GREEN,
+                    label="Click Me",
+                    disabled=False,
+                )
+            )
+        ]
+
+        await channel.send("Look a Button!", components=components)
+
+    @component_callback("my_button")
```

Comment:
I'd put comment here mentioning that the parameter is the parameter is the custom ID

Reply:
maybe to be clearer, also change `"my_button"` to `"my_button_id"`